install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ $(whoami) != "root" ]]; then
+  echo "This file needs to be executed as root or with sudo."
+  exit
+fi
+
+defaultdir="/usr/bin"
+
+echo "Directory to install to (default: $defaultdir)"
+read installdir
+
+if [[ $installdir = "" ]]; then
```

Comment:
This syntax can be simplified to 
```bash
[ -z "$installdir" ] && installdir=$defaultdir
```

This can be done for other if statements true, where a condition is ran in [ ], then a logical operator is used, such as && where if the first one is true, the second one is ran. Or ||, where if the first one fails or is false, then the next one is ran. This is cleaner and neater syntax which will also make the script smaller. If you don't understand this I can make these changes so you can see what I mean.